tests/app/dao/test_ft_billing_dao.py

```diff
@@ -592,10 +592,22 @@ def test_fetch_sms_billing_for_all_services_with_remainder(notify_db_session):
 
     results = fetch_sms_billing_for_all_services(datetime(2019, 5, 1), datetime(2019, 5, 31))
     assert len(results) == 3
-    assert results[0] == (org.name, org.id, service.name, service.id, 10, Decimal('0.11'), 8, 3, 0, Decimal('0'))
+
+    # Field names are: organisation_name, organisation_id, service_name, service_id, free_sms_fragment_limit,
```

Comment:
Ahh that's a good suggestion 👍🏼 . I will improve the tests and let you know when I'm done.

Reply:
Hey @benthorner I took a stab at changing the tests, they look to be failing because we are comparing a tuple to odict_values(). Do you have an idea for an elegant way to solve this, or should I try a loop of assertions or something? Feel free to add a commit if you find something that works ☺️

Reply:
@CrystalPea I had a quick play. You could do this:

```
expected_results = [
  { 'organisation_name': org.name, ... },
  ...
]

assert [dict(result) for result in results] == expected_results
```

Basically the same idea, but converting each result to a `dict` first so there aren't any type issues.

Thanks for taking the time to try and refactor. Let me know if this doesn't work and you need a hand.

Reply:
This works, thanks a lot @benthorner ! I think this PR is ready to go now ^_^